core/src/components/progress-bar/progress-bar.scss

```diff
@@ -10,7 +10,7 @@
   * @prop --progress-background: Color of the progress bar
   * @prop --buffer-background: Color of the buffer bar
   */
-  --background: #{ion-color(primary, base, 0.3)};
+  --background: #{ion-color(primary, base, 0.2)};
```

Comment:
Why did the opacity get changed back here?

Reply:
It's just from the version 5.5.4 (latest tag). I started with this version. In my first PR I changed it to 0.3 because it's slightly better to see in both themes. But idk. Change it if you want.

Reply:
Ah ok, I will change it back. 0.3 seemed to better match the progress bar spec for Material Design.